scripts/gen-dockerfile.sh

```diff
@@ -13,6 +13,8 @@ WORKDIR /tikv
 
 # Install Rust
 COPY rust-toolchain ./
+RUN rustup self update
+RUN rustup set profile minimal
```

Comment:
what did this for

Reply:
Speeds up `docker build .` by using rust minimal profile.

https://blog.rust-lang.org/2019/10/15/Rustup-1.20.0.html#profiles